docs/develop/func/cookbook.md

```diff
@@ -505,19 +512,19 @@ send_raw_message(msg, 3); ;; mode 3 - pay fees separately and ignore errors
 
 > 💡 Noted
 >
-> TODO: what is literal `a` and were devs can find more information about literal `a`?
+> In this example, we use literal `a` to get address. You can find more about string literals in [docs](/docs/develop/func/literals_identifiers#string-literals)
 
 > 💡 Noted
 >
-> TODO: what is mode `3` and which preffer mode to use in most cases? When we should use mode 64 and 128?
+> In this example, we used mode 3 to take the incoming tones and send exactly as much as specified (amount) while paying commission from the contract balance and ignoring the errors. Mode 64 is needed to return all the tones received, subtracting the commission, and mode 128 will send the entire balance.
```

Comment:
"Tones" goes a bit against of brand assets https://ton.org/en/brand-assets, which state that valid name for this cryptocurrency are "Toncoin" and "TON".

Reply:
This is a typo.